Python/generated_cases.c.h

```diff
@@ -689,25 +689,28 @@
         }
 
         TARGET(RAISE_VARARGS) {
+            PyObject **args = &PEEK(oparg);
             PyObject *cause = NULL, *exc = NULL;
             switch (oparg) {
             case 2:
-                cause = POP(); /* cause */
+                cause = args[1];
                 /* fall through */
             case 1:
-                exc = POP(); /* exc */
+                exc = args[0];
                 /* fall through */
             case 0:
                 if (do_raise(tstate, exc, cause)) {
-                    goto exception_unwind;
+                    if (true) { STACK_SHRINK(oparg); goto exception_unwind; }
                 }
                 break;
             default:
                 _PyErr_SetString(tstate, PyExc_SystemError,
                                  "bad RAISE_VARARGS oparg");
                 break;
             }
-            goto error;
+            if (true) { STACK_SHRINK(oparg); goto error; }
+            STACK_SHRINK(oparg);
+            DISPATCH();
```

Comment:
The generator emits some unreachable code here. If we had an unconditional ERROR macro we could avoid that.

Reply:
The generator could special-case `true` as the argument. There are already other things it checks for to skip generating such dead code (mostly `return` or `goto` or macros containing those), we could add this.

Do you feel it's important enough to do it now? The C compiler or LTO will most likely delete that code anyway, so it doesn't really cost much.